app/javascript/components/key-performance-indicators/components/date-range-dialog/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
```

Comment:
Extract this code to a file called components.jsx and in this file export that component (`export { default } from "./component"`)